src/app/pages/Search/Search.tsx

```diff
@@ -1,28 +1,22 @@
 import React, { useState } from "react";
+import { GitHubData, TransformedResult } from "../../../types";
 import FooterMenu from "../../components/FooterMenu/FooterMenu";
 import HeaderSearch from "../../components/HeaderSearch/HeaderSearch";
-import SearchResults from "../../components/SearchResults/SearchResults";
+import SearchResultsComponent from "../../components/SearchResults/SearchResults";
 import styles from "./Search.module.css";
 
-type GitHubData = {
-  items: [{ name: string; html_url: string }];
-};
-
-export type TransformedResult = {
-  name: string;
-  rawUrl: string;
-};
-
 function Search(): JSX.Element {
   const [searchResults, setSearchResults] = useState<TransformedResult[]>([]);
 
   function fetchGitHubData(searchValue: string, filterValue: string) {
     fetch(`/api/search?code=${searchValue}&user=${filterValue}`)
       .then((response) => response.json())
       .then((data: GitHubData) => {
-        const transformed = data.items.map((item) => {
+        const transformed: TransformedResult[] = data.items.map((item) => {
           return {
             name: item.name,
+            repoName: item.repository.full_name,
```

Comment:
GitHub has a different naming convention `full_name` instead of `fullName`.
I would recommend transforming the data on your server to have a consistent naming convention on client.